DirectProgramming/DPC++/CombinationalLogic/sepia-filter/sample.json

```diff
@@ -0,0 +1,45 @@
+{
+  "guid": "B9C425DB-A3AD-4FCB-9CA0-1909E5189FB7",
+  "name": "Sepia Filter",
+  "categories": ["Toolkit/Intel® oneAPI Base Toolkit/oneAPI DPC++ Compiler/CPU and GPU"],
+  "toolchain": ["dpcpp"],
+  "description": "A program that converts an image to sepia tone",
+  "languages": [{"cpp":{}}],
+  "targetDevice": ["CPU", "GPU"],
+  "os": ["linux", "windows"],
+  "builder":["ide", "cmake"],
+  "ciTests": {
+    "linux": [
+      {
+	"id": "build", 
+	"steps": [
+	  [ "mkdir build" ],
+	  [ "cd build" ],
+	  [ "cmake .." ],
+	  [ "make" ]
+	]
+      },
+      {
+	"id": "run",
+	"steps": [
+	  [ "make run" ]
+	]
+      } 
```

Comment:
Individual objects with `id` field mostly serve to distinguish different build targets, e.g. "cpu", "gpu", "fpga_emu". There's no distinction between "build" and "run" stages anymore, you can just put arbitrary commands under `steps`. Also, `steps` is an array of strings, not an array of arrays.
So, for your sample your CI tests for Linux would be:
```
  "linux": [
      {
	"steps": [
	  "mkdir build",
	  "cd build",
	  "cmake ..",
	  "make",
	  "make run",
	]
      }
   ]
```
Also, if you have the "clean" target available, please also add it as the last step.
Please, make corresponding changes for Windows.

Reply:
Thanks, Roman. Updated sample.json